synapse/handlers/room.py

```diff
@@ -370,11 +370,12 @@ def clone_existing_room(
         if current_power_level < needed_power_level:
             # Perform a deepcopy in order to not modify the original power levels in a
             # room, as its contents are preserved as the state for the old room later on
-            new_power_levels = copy.deepcopy(power_levels)
-            initial_state[(EventTypes.PowerLevels, "")] = new_power_levels
-
-            # Assign this power level to the requester
+            new_power_levels = {k: v for k, v in power_levels.items() if k != "users"}
```

Comment:
A comment to say that we're working around the fact this could be a frozendict would be good, I got quite confused.